webapp/components/Navbar/Navbar.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <nav>
+    <div class="logo">
+      <NuxtLink to="/">
+        <Logo />
+        news
+      </NuxtLink>
+    </div>
+    <div v-if="!isAuthenticated" class="option">
+      <NuxtLink to="/login">Login</NuxtLink>
+    </div>
+    <div v-if="!isAuthenticated" class="option">
+      <NuxtLink to="/signup">Signup</NuxtLink>
+    </div>
+    <div v-if="isAuthenticated" class="option">
+      <a id="a_logout" @click="() => $emit('logout')">Logout</a>
+    </div>
+  </nav>
+</template>
+
+<script>
+import Logo from '@/components/Logo/Logo'
+export default {
+  name: 'Navbar',
```

Comment:
⭐ ⭐ For a menu component which shows a login or logout button and its software tests.